sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/fcm/RegistrarClient.java

```diff
@@ -158,36 +157,37 @@ public boolean unregister(String skypeUserToken) {
 
         CountDownLatch latch = new CountDownLatch(1);
 
+        final Throwable[] requestError = { null };
```

Comment:
Any reason this is an array instead of a variable?

Reply:
A variable is not accessible inside below anonymous class without final.
But a final variable is not editable.
So only a final array is bot accessible and editable inside below anonymous class new HttpCallback().
This wrapper is suggested by Android Studio but quite a bit strange to me as well.....